articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+
+***Transparency:*** Before an existing version is changed, everyone in the blockchain network must agree since the records are shared. An overhaul of the whole network would be required to change a single transaction on a blockchain network, making the data on the network transparent, consistent and largely accurate. All participants can access any transaction on the blockchain as long as they have a private key, as every transaction is decentralized. This makes it easier to share information across multiple platforms.
+
+### Implementing Blockchain in health care
+![Health care img](/engineering-education/blockchain-as-a-revolution-inhealthcare/Healthcare-img.jpg)
+[*Image Source: https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470*](https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470)
+
+There are numerous applications of Blockchain in healthcare:
+
+### Record Keeping
+***Medical History Tracking***
+With Blockchain, patients' treatment history can be tracked anywhere and at any time with no ambiguity, hence providing health practitioners a better context.
+
+_In Data Security: [Do you know that your medical data is worth more than your credit card number in the black market?](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561) According to a report by Protenus Breach Barometer, about 140 million patient health records were hacked between 2015 and 2016._
+>**Blockchain has the potential to keep private information safe and secure while still being accessible anywhere.**
+
+***Medical Device Usage Tracking***
+The use of  medical devices such as electronic wearables have been on the rise, with new reports revealing that forty percent of medical patients use such devices to track their health. However, this makes it difficult to store all the  daily medical reports from devicse for each patients. This is another area where keeping blockchain records can help.
+
+***Drug Supply Chain Tracking***
+According to Hudson River Field Office (HRFO), over fifteen percent of drugs sold in developing countries are counterfeit. In addition, pharmaceutical companies lose about $200 billion annually due to counterfeiting. Blockchain can curb this trend with features like point-by-point, which tracks possibilities and proof of authenticity. This allows users to verify drug authenticity before purchase or consumption.
+
+### Digital Identity
+***Patient Identity Validation***
+Blockchain can solve the issue of patient identity theft and can reduce the use of fake IDs by impersonators.
+
+***Healthcare Provider Identity Validation***
+Impersonation using fake medical records is a common problem in third world countries and has caused great number of deaths. Blockchain can address these issues with its validation system.
```

Comment:
Give specific examples. Which third world country? How do you know this is a common problem? Stats and/or sources needed.

Reply:
Removed point.